txnbuild/operation_test.go

```diff
@@ -214,8 +214,8 @@ func TestSetOptionsFromXDR(t *testing.T) {
 	var opSource xdr.AccountId
 	err := opSource.SetAddress("GB7BDSZU2Y27LYNLALKKALB52WS2IZWYBDGY6EQBLEED3TJOCVMZRH7H")
 	assert.NoError(t, err)
-	cFlags := xdr.Uint32(5)
-	sFlags := xdr.Uint32(7)
+	cFlags := xdr.Uint32(0xD)
+	sFlags := xdr.Uint32(0xF)
```

Comment:
Nit, for readability.
```suggestion
	cFlags := xdr.Uint32(0b1101)
	sFlags := xdr.Uint32(0b1111)
```